openfasoc/generators/lc-dco/simulations/Ngspice/LC_run_lvt.netlist

```diff
@@ -3,7 +3,7 @@
 
 .param mc_mm_switch=0
 .param mc_pr_switch=0
-.lib ../../PDK/open_pdks/sky130/sky130A/libs.tech/ngspice/sky130.lib.spice tt
+.lib ../../../PDK/open_pdks/sky130/sky130A/libs.tech/ngspice/sky130.lib.spice tt
```

Comment:
@leochand101 Can you try to change those to absolute paths

Reply:
I am not clear as to where these are being referred to inside the OpenFASOC folder I think based on Sai's updates these were being being installed /usr/local/share/pdk/sky130A/ . I updated it to the same.

Reply:
@leochand101 I think that comes from the $PDK_ROOT. You can use that. According to the installation script, this variable is /usr/bin/miniconda3/share/pdk (check here https://github.com/idea-fasoc/OpenFASOC/blob/9e5f6ef6b2106e2a22511237c205bc323462ae86/dependencies.sh#L135)